flink-libraries/flink-state-processing-api/src/test/java/org/apache/flink/state/api/RocksDBStateBackendReaderKeyedStateITCase.java

```diff
@@ -22,12 +22,9 @@
 import org.apache.flink.runtime.state.StateBackend;
 import org.apache.flink.runtime.state.memory.MemoryStateBackend;
 
-import org.junit.Ignore;
-
 /**
  * IT Case for reading state from a RocksDB keyed state backend.
  */
-@Ignore
```

Comment:
How is this test related to the change?

Reply:
@rmetzger disabled that test yesterday so that travis would not spuriously fail other PRs. I'm re-enabling the test as part of the fix.

Reply:
Ah, just saw the discussions in the JIRA. Got it, seems like a revert of temporary ignore.